User-Service-Test/ServiceTests/UserControllerTest.cs

```diff
@@ -0,0 +1,43 @@
+using AutoMapper;
+using Castle.Core.Logging;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Moq;
+using User_Service.Controllers;
+using User_Service.Interfaces;
+using User_Service.Models;
+using Xunit;
+
+namespace User_Service_Test.ServiceTests
+{
+    public class UnitTeUserControllerTestst1
```

Comment:
I think something might have gone wrong in the naming here. Did you mean UnitTestUserController (or more specifically UnitTestOrganisationController)?